sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/mSers/MSersDirectoryPage.java

```diff
@@ -18,11 +18,24 @@ public class MSersDirectoryPage {
   public static final By DELETE_ICON = By.xpath("(//div[@class=\"component-wrap\"]//div)[1]");
   public static final By EDIT_ICON = By.xpath("(//div[@class=\"component-wrap\"]//div)[1]");
   public static final By DISEASE_COMBOBOX = By.cssSelector("#disease div");
+  public static final By POINT_OF_ENTRY_COMBOBOX = By.cssSelector("#pointOfEntry div");
+  public static final By FACILITY_COMBOBOX = By.cssSelector("#healthFacility div");
+  public static final By DISTRICT_COMBOBOX = By.cssSelector("#district div");
+  public static final By REGION_COMBOBOX = By.cssSelector("#region div");
+  public static final By GROUPING_COMBOBOX = By.xpath("(//div[@class='v-filterselect-button'])[1]");
+  public static final By GROUPING_COMBOBOX_INPUT =
+      By.xpath("(//div[@class='v-filterselect-button'])[1]/..//preceding-sibling::input");
+  public static final By SHOW_ROWS_FOR_DISEASES_LABEL =
+      By.xpath("//label[text()=\"Show 0-rows for disease(s)\"]");
```

Comment:
can we create it without using text? in this way we can use it also for german environment

Reply:
done